src/encoded/audit/experiment.py

```diff
@@ -1904,6 +1904,24 @@ def audit_experiment_gtex_biosample(value, system):
     return
 
 
+@audit_checker('Experiment', frame=['object'])
```

Comment:
I know, but I don't see another way of doing that. Except comparing with the list of submitted by me experiments. I think this way we will become aware of anything not submitted. And once we don't have any more audits of this type - we can make a swap comparing with the list of already submitted by me.